cmd/ci-secret-generator/main.go

```diff
@@ -367,14 +400,16 @@ func main() {
 		if err != nil {
 			logrus.WithError(err).Fatal("failed to create dry-run mode client")
 		}
-	} else {
+	case targetBitwarden:
 		var err error
 		client, err = bitwarden.NewClient(o.bwUser, o.bwPassword, func(s string) {
 			secrets.Insert(s)
 		})
 		if err != nil {
 			logrus.WithError(err).Fatal("failed to create Bitwarden client")
 		}
+	default:
```

Comment:
Let's not panic

Reply:
Hmm, my impression was it is the `assert` equivalent in Go.  That is an unreachable path unless there is a logic error in the program, what would be a better alternative?

Reply:
Not sure what exactly assert is, but the only places that are ok for panics are:
* func main
* func init
* Package-level variable declarations
* Testcode

It doesn't really make much of a difference there if the reason for the error is the programmers fault or someone elses IMHO.